Added information about maximum payload size (#459)
Haivision/srt: docs/live-streaming.md

Note that SRT isn't limited to MPEG-TS -- it can be applied to any "live streaming" data 
transmission (as long as you use Live mode, which is the SRT default mode). You can use 
any other suitable data format, and any intermediate protocol on top of MPEG-TS with 

> an extra header (if you want to do this - as people often try this with RTP - note

Reviewer: ```suggestion
an extra header (this is an option that people often try with RTP) - note
```

> that 1316 is the default maximum payload size, which can be changed using

Reviewer: ```suggestion
that 1316 is the default maximum payload size, which can be changed using the
```
